tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -928,29 +934,116 @@ def topic_partitions_on_pod():
+        ])
+
+        self.logger.info('patching deployment to allow downscaling')
+        deployment_args = deployment_args.replace('--allow-downscaling=false',
+                                                  '--allow-downscaling=true')
+        patch = [{
+            'op': 'replace',
+            'path': '/spec/template/spec/containers/0/args',
+            'value': json.loads(deployment_args)
+        }]
+        patch_str = json.dumps(patch)
+        # kubectl patch deployment redpanda-controller-manager -n=redpanda-system --type=json \
+        #         --patch='[{"op": "replace", "path": "/spec/template/spec/containers/0/args", "value": ["arg1", "arg2", "etc..."]}]'
+        self.redpanda.kubectl.cmd([
+            'patch', 'deployment', 'redpanda-controller-manager',
+            '-n=redpanda-system', '--type=json', f'-p={patch_str}'
+        ])
+
+        custom_resource = f'redpandas/rp-{self.redpanda.cluster_id}'
+        self.logger.info(
+            'getting replicas from custom resource {custom_resource}')
+        # kubectl get redpandas/rp-clkd0n22nfn1jf7vd9t0 -n=redpanda -o=jsonpath='{.spec.clusterSpec.statefulset.replicas}'
+        replicas = self.redpanda.kubectl.cmd([
+            'get', custom_resource, '-n=redpanda',
+            "-o=jsonpath'{.spec.clusterSpec.statefulset.replicas}'"
+        ])
+
+        orig_replicas = int(replicas)
+        new_replicas = orig_replicas - 1
+        self.logger.info(
+            'decomm by patching custom resource {custom_resource} with replicas {new_replicas}'
+        )
+        patch = [{
+            'op': 'replace',
+            'path': '/spec/clusterSpec/statefulset/replicas',
+            'value': new_replicas
+        }]
+        patch_str = json.dumps(patch)
+        # kubectl patch redpandas/rp-clkd0n22nfn1jf7vd9t0 -n=redpanda --type=json -p='[{"op": "replace", "path": "/spec/clusterSpec/statefulset/replicas", "value": 4}]'
+        self.redpanda.agent_ssh([
+            'patch', custom_resource, '-n=redpanda', '--type=json',
+            f'-p={patch_str}'
+        ])
+
+        self.logger.info('waiting for decommissioning to complete')
+        wait_until(lambda: cr_ready_status(custom_resource) == 'True',
+                   timeout_sec=600,
+                   backoff_sec=1)
+
+        pvc_name = f'datadir-rp-{self.redpanda.cluster_id}-{new_replicas}'
+        self.logger.info('deleting pvc {pvc_name}')
+        # kubectl delete pvc datadir-rp-clkd0n22nfn1jf7vd9t0-4 -n=redpanda
+        self.redpanda.kubectl.cmd(['delete', 'pvc', pvc_name, '-n=redpanda'])
+
+        # skip decomm via broker admin api so it does not conflict with decomm via kubectl
+        #admin = self.redpanda._admin
+        #admin.decommission_broker(pod_id)
+        #waiter = NodeDecommissionWaiter(self.redpanda, pod_id, self.logger, progress_timeout=120)
+        #waiter.wait_for_removal()
+        #self.redpanda.stop_node(pod)
         assert topic_partitions_on_pod() == 0
```

Comment:
from discussion with @savex : may need to adjust this line based on how kubectl is removing/adding pods